.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 __pycache__
 .cache
+ulauncher-statuspages
```

Comment:
Why this?
Have you run the "make link" command for development or did you create the symlink to ulauncher extensions directory yourself? That directory should never be created. It happened to me before, generating a symlink in the same folder of the extension which resulted in that folder being created.

Reply:
This directory was occasionally generated by `make link`.